crates/wash-cli/src/ctl/output.rs

```diff
@@ -84,7 +86,10 @@ pub fn links_table(list: Vec<InterfaceLinkDefinition>) -> String {
 }
 
 /// Helper function to transform a Host list into a table string for printing
-pub fn hosts_table(hosts: Vec<Host>) -> String {
+pub fn hosts_table(mut hosts: Vec<Host>) -> String {
+    // Sort hosts by uptime_seconds in descending order
+    hosts.sort_by(|a, b| b.uptime_seconds.cmp(&a.uptime_seconds));
```

Comment:
I figured it would enhance visibility and make it easier to quickly understand the deployment order of the hosts. Plus, displaying them based on uptime just looks more uniform and consistent.
But I'd be happy to modify it to friendly_name.